src/container/App.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './App.css';
+import Display from '../components/Display/Display';
+import ButtonPanel from '../components/ButtonPanel/ButtonPanel';
+import calculate from '../logic/calculate';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      calculation: '0',
+      total: '',
+      current: '',
+      operator: ''
+    }
+  }
+
+
+  render() {
+
+    const handleClick = (btnName) => {
```

Comment:
This `handleClick` function should be part of the `App` 's prototype. Therefore, it is more proper to place it as an instance method outside of this `render` function. Please make this change accordingly.